components/proxy/src/main/java/com/hotels/styx/admin/AdminServerBuilder.java

```diff
@@ -200,6 +201,8 @@ private HttpHandler adminEndpoints(StyxConfig styxConfig, StartupConfig startupC
         httpRouter.aggregate("/admin/servers", serverHandler);
         httpRouter.aggregate("/admin/servers/", serverHandler);
 
+        httpRouter.aggregate("/metrics", new PrometheusHandler(environment.meterRegistry()));
```

Comment:
We still have `/admin/metrics` interface. What does it show now with the Prometheus metrics?

Reply:
`/metrics` was chosen because it's the default path for prometheus metrics. `the old endpoint still works but will only show the older Dropwizard meters.